CSS/index.css

```diff
@@ -0,0 +1,197 @@
+body {
+    margin: 0;
+    background-color: rgb(50, 50, 50);
+    color: white;
+    font-family: "Playfair Display", Verdana, Geneva, Tahoma, sans-serif;
+}
+
+h1 {
+    text-align: center;
+    color: #eb8d46;
+}
+
+h2 {
+    color: #eb8d46;
+}
+
+main {
+    padding: 120px 20px;
+}
+
+a {
+    color: #3f5db7;
+    font-style: italic;
+}
+
+section {
+    padding: 10px 20px;
+    background-color: rgb(50, 50, 50);
+    border: black solid 5px;
+    margin: 20px 5px;
+
+}
+
+.imgMain {
+    display: block;
+    margin-top: 20px;
+    margin-left: auto;
+    margin-right: auto;
+    width: 500px;
+    border: 20px solid #3f5db7;
+}
+
+/* Tablet */
+
+@media (max-width:1024px) {
+   body {
+    font-size: .9rem;
+   }
+
+   .imgMain {
+    margin-top: 0px;
+    width: 350px;
+   }
+
+   main {
+    padding: 100px 20px;
+   }
+}
+
+
+/* Mobile */
+
+@media (max-width:430px) {
+    body {
+     font-size: .81rem;
+    }
+ 
+    .imgMain {
+     margin-top: 0px;
+     width: 150px;
+    }
+ 
+    main {
+     padding: 100px 10px;
+    }
+
+    h1 {
+     font-size: 1.5rem;   
+    }
+ }
+
+/* Text Modifiers */
+
+.boldFont {
+    font-weight: bolder;
+}
+
+/* Header */
+
+nav {
+    display: flex;
+    gap: 10px;
+    justify-content: flex-end;
+    margin: 30px;
+}
+
+.navItem {
+    color: #3f5db7;
+    background-color: white;
+    padding: 10px 25px;
+    font-style: normal;
+    font-size: 1.25rem;
+    text-decoration: none;
+    font-weight: bold;
+}
+
+.navItem:link, .navItem:visited {
+    background-color: white;
+}
+
+.navItem:focus, .navItem:hover {
+    background-color: #eb8d46;
+}
+
+.navItem:active {
+    background-color: rgb(255, 123, 0);
+}
+
+header {
+    background-color: black;
+}
+
+.sticky {
+    position: fixed;
+    top: 0;
+    width: 100%
+}
+
+/* Tablet Header */
+
+@media (max-width:1024px) {
```

Comment:
Again, a bit hard to see what your queries were visually doing to the screen, but they do work.

Reply:
Resolved